.zombienet-tests/utils.ts

```diff
@@ -0,0 +1,8 @@
+import { createClient } from "../packages/substrate-client/dist"
+import { ScProvider } from "../packages/sc-provider"
+
+export async function connect(networkInfo: any) {
+  const customChainSpec = require(networkInfo.chainSpecPath)
+  let provider = ScProvider(JSON.stringify(customChainSpec))
+  return createClient(provider)
+}
```

Comment:
This file is very simple and it seems that it could be a plain JS file.
That way there is no need to `tsc file.ts || true` in the github action workflow file

Reply:
All scripts in zombienet are typescript and are read from the `ts-script` parser command. I think it is better to keep all of them in `ts` than having some in `ts` and some in `js`, but 🤷🏽

Reply:
Suggestion was meant to simplify the workflow file and local testing.

Using this `utils.ts` file requires the user to do `tsc utilt.ts` before executing the zombienet test locally